src/content/docs/en/recipes/call-endpoints.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Call endpoints from the server
+description: Learn how to call endpoints from the server in Astro.
+i18nReady: true
+type: recipe
+---
+
+## Recipe
+
+1. Create an endpoint (e.g. `src/pages/api/hello.ts`):
+
+    ```ts title="src/pages/api/hello.ts"
+    import type { APIRoute } from 'astro'
+
+    export const get: APIRoute = () => {
+      return {
+        body: JSON.stringify({
+          hello: 'world',
```

Comment:
```suggestion
          greeting: 'Hello',
```
So, I didn't like that "world" is not a reasonable value for "hello"  ??  :sweat_smile: Plus, since you use this to recreate the text "Hello world" it's not great to have duplicating words because it's not as clear tracing through that things are entirely separate.